demos/node/import/index.js

```diff
@@ -2,16 +2,12 @@
 
 /* eslint-disable no-console */
 
-import Vex from '../../../build/vexflow-debug.js';
+import Vex from '../../../build/vexflow.module.js';
```

Comment:
The current 4.0.0-alpha branch creates a `vexflow.module.js` that has an `export default Vex;` at the bottom.